### DISCUSSION TITLE
Delete all edit states.

### DISCUSSION THREAD

**Discussion by iOSyan** [hm21/pro_image_editor#80] [Q&A] [answered]
Is there a way to delete all edit states with one click? I want to clear the entire canvas.

**Comment by hm21** [comment, marked as answer]
This was not possible, but I have released version `3.0.11` for you which includes this feature.

You can call it like this: `editorKey.currentState!.removeAllLayers();`.

**Comment by iOSyan** [comment]
It's run well! Thank you so much!
